charts/astronomer/templates/commander/commander-rolebinding.yaml

```diff
@@ -9,6 +9,7 @@
 {{- $useClusterRoles := and .Values.global.rbacEnabled .Values.global.clusterRoles (not .Values.global.features.namespacePools.enabled) }}
 {{- $shouldCreateResources := and .Values.global.rbacEnabled (or (and .Values.global.clusterRoles (not .Values.global.features.namespacePools.enabled )) $namespacepoolsRBACresources) }}
 {{- $namespaces := .Values.global.features.namespacePools.namespaces.names }}
+{{- $data := ( include "commander.ServiceAccount" . )  -}}
```

Comment:
change this to commanderServiceAccount as variable declaration

Reply:
done